digitalocean/resource_digitalocean_loadbalancer.go

```diff
@@ -122,6 +122,16 @@ func resourceDigitalOceanLoadBalancerV0() *schema.Resource {
 					return strings.ToLower(val.(string))
 				},
 			},
+			"size": {
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
In my testing, attempting to resize a load balancer fails. If that is the intended behavior, we should set `ForceNew: true`.

```
Changes to Outputs:
  ~ size = "lb-small" -> "lb-large"

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

digitalocean_loadbalancer.public: Modifying... [id=d633a577-2fd9-4cb8-a3bc-e88591ddcc49]

Error: Error updating Load Balancer: PUT https://api.digitalocean.com/v2/load_balancers/d633a577-2fd9-4cb8-a3bc-e88591ddcc49: 422 (request "2ac0de42-5470-45a8-b127-1e4687cbf85c") size cannot be modified
```

Reply:
That's correct; at the moment, changing the size is not allowed. I'll add `ForceNew: true` with a note that it should be removed when resizing is allowed